tests/test_adjoint_jacobian.py

```diff
@@ -645,7 +645,7 @@ def circuit_ansatz(params, wires):
     qml.Rot(params[6], params[7], params[8], wires=wires[0])
     # #     qml.Rot(params[8], params[8], params[9], wires=wires[1]).inv()
```

Comment:
Why do we have this commented line?

Reply:
To be clear, it's not commented in this PR! :wink: The inverse of `qml.Rot` is not fully supported in Lightning the last time I checked and so it was removed from `test_integration` too. I'll double-check and if this is so I'll update the test consequently.